pypesto/optimize/io.py

```diff
@@ -0,0 +1,221 @@
+"""Load/reconstitute results from history."""
```

Comment:
wouldn't it be more consistent to also have this in `pypesto/result/load.py` or even `pypesto/result/optimize.py`. Technically no `o` here, only `i`, so I would still consider renaming even if it stays in this dir.

Reply:
yeah, one could also move all io functionality to store/result. will leave for the moment as it does not disturb, renaming to "load.py" (was called that before actually).